test/unit/util/test_paths.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_in_directory():
-    base_dir = tempfile.mkdtemp()
+    base_dir = os.path.realpath(tempfile.mkdtemp())
```

Comment:
What about also use [tmp_path fixture](https://docs.pytest.org/en/stable/how-to/tmp_path.html) instead of `mkdtemp()`, so that removal of temporary files is taken care of?
Not sure if you'd still need the `os.path.realpath()`, but it'd still be a worthy improvement.